release_task.rb

```diff
@@ -184,4 +186,11 @@ def define_version_update_task
       end
     end
   end
+
+  def define_release_task
```

Comment:
Although I couldn't come up with the better name, I feel like this method name include all tasks in this class because the class name is ReleaseTask too.

Reply:
Like `definde_batch_release_tasks`?
Wouldn't that fit with the other naming rules?